internal/services/aws_reservation_service.go

```diff
@@ -128,6 +128,7 @@ func CreateAWSReservation(w http.ResponseWriter, r *http.Request) {
 		// Direct AMI or no image were provided (launch template), no need to call image builder
 		ami = reservation.ImageID
 	} else {
+		// Not prefixed with "ami-" therefore this must be a valid UUID
```

Comment:
Could we just include validation here? I mean we do that for all the other providers, so why not here and the feedback for user would be immediate and more accurate, Invalid request - "the imageID must be UUID"